src/Domain/FunctionsOfTime/FunctionOfTime.hpp

```diff
@@ -55,6 +55,12 @@ class FunctionOfTime : public PUP::able {
   /// interval.
   virtual std::array<double, 2> time_bounds() const = 0;
 
+  /// The smallest value `time_bounds()[1]` could have that is greater
+  /// than \p time.  For updatable functions of time, this will be the
+  /// expiration time of the last update before or at \p time.  For
+  /// non-updatable functions, this will be infinity.
```

Comment:
I found this description a bit confusing because it was too compressed. What about something like

```
\brief The first expiration time after \p time.

\details For non-updatable functions, this returns infinity. For updatable
functions, the first expiration time after \p time is found by determining the
update immediately before \p time. The expiration time of this update is what
is returned. If \p time happens to be an update itself, then the expiration of
that update is returned
```